src/collectd.conf.in

```diff
@@ -50,6 +50,17 @@
 #WriteQueueLimitHigh 1000000
 #WriteQueueLimitLow   800000
 
+#----------------------------------------------------------------------------#
+# StartRead specifies when the first read for each read plugin is performed. #
+# The float value is in seconds and specifies the minutes, seconds and       #
+# milliseconds per hour. This may be overwritten on a per-plugin base by     #
+# using the 'StartRead' option of the LoadPlugin block:                       #
+#   <LoadPlugin foo>                                                         #
+#       StartRead 301.42 # minute 5, second 1, millisecond 420               #
+#   </LoadPlugin>                                                            #
+#----------------------------------------------------------------------------#
+#StartRead -1.0
```

Comment:
1) This is not "StartRead" but "DelayRead"
2) Negative value has no sence.

Reply:
1. Maybe "StartRead" was not the best choice. The value/option delays only the first read. So "DelayRead" may also be misleading. However, I do not insist on "StartRead"
2. Zero has a specific meaning: start reading at second 0 of each minute. This looks a lot nicer in timelines than starting at an arbitrary second.
3. Milliseconds enable explicit ordered execution of multiple plugins within a second, but avoid parallel reads which may affect each other. And we can still get the same epoche timestamp for all read values. (most reads are extremely fast, significantly below a second.

Reply:
>Zero has a specific meaning: start reading at second 0 of each minute. 

1) I don't found this documented in provided PR
2) Typical Collectd setup calls read callbacks with 10 seconds interval. Once per minute is not enough frequency.
3) When there is not enough ReadThreads, this will not work.

>This looks a lot nicer in timelines than starting at an arbitrary second.

arbitrary second? Strange wish...

Reply:
Milliseconds enable explicit ordered execution of multiple plugins within a second, but avoid parallel reads which may affect each other. And we can still get the same epoche timestamp for all read values. (most reads are extremely fast, significantly below a second.

From one side you want to serialize  read callbacks, from other side you want to get same epoch timestamp. contradiction.

>but avoid parallel reads which may affect each other.

Example, use-case?